include/clang/AST/Type.h

```diff
@@ -3357,6 +3379,8 @@ class FunctionProtoType : public FunctionType, public llvm::FoldingSetNode {
     }
     if (hasExtParameterInfos())
       EPI.ExtParameterInfos = getExtParameterInfosBuffer();
+    EPI.ParamBounds = hasParamBounds() ? param_bounds_begin() : nullptr;
+    EPI.ReturnBounds = getReturnBounds();
```

Comment:
I'll update this for consistency.  If we ever change the representation of ReturnBounds, we'll want this pattern.